src/features/programSample/ProgramSampleModal.tsx

```diff
@@ -435,17 +437,20 @@ const ProgramModem = ({
             logger.info(
                 `${json.step}/${json.amountOfSteps}: ${json.progressPercentage}% - ${json.message}`
             );
-            const amountOfProgress =
-                // Future api of shared will remove undefined steps
-                ((json.step! - 1) / json.amountOfSteps!) * 100 +
-                (1 / json.amountOfSteps!) * json.progressPercentage;
 
-            memoizedProgress =
-                amountOfProgress > memoizedProgress
-                    ? amountOfProgress
-                    : memoizedProgress;
+            if (json.step && json.amountOfSteps) {
+                const amountOfProgress =
+                    // Future api of shared will remove undefined steps
+                    ((json.step - 1) / json.amountOfSteps) * 100 +
+                    (1 / json.amountOfSteps) * json.progressPercentage;
```

Comment:
Same thing as Jonas said for the other instance of amountOfProgress above.
Also it most definitely is given to you by the nrfutil wrapper now.